src/entities/Deal.ts

```diff
@@ -21,15 +25,37 @@ export class Deal {
   private initializedPromise: Promise<void>;
   private subscriptions = new DisposableCollection();
 
+  public get registration(): IDealConfig {
+    return this.rootData.registration;
+  }
+
+  // public get votes(): Array<IVoteInfo> {
+  //   return this.rootData.votes;
+  // }
+
+  // public get discussions(): Array<Array<IClause, Hash>> {
+  //   return this.rootData.discussions;
+  // }
+
+  public get isOpen(): boolean {
+    return this.rootData?.registration.daos.length === 1;
```

Comment:
This is just an assumption I'm making about how to determine "open deals".
Should be replaced when we have a stable schema